juego.py

```diff
@@ -237,19 +257,32 @@ def play(self, level):
             pygame.display.update()
 
     def run(self):
-        pygame.init()
+        self.running = True
+        pygame.font.init()
+        pygame.mixer.init()
```

Comment:
all pygame module inits should be in the sugargame.canvas.PygameCanvas call

Reply:
Done in 1875082